x/feemarket/post/fee.go

```diff
@@ -107,7 +108,7 @@ func (dfd FeeMarketDeductDecorator) PostHandle(ctx sdk.Context, tx sdk.Tx, simul
 	}
 
 	ctx.Logger().Info("fee deduct post handle",
-		"fee", feeCoins,
+		"fee", payCoin,
```

Comment:
`DeductFeeAndTip()` and `state.Update` are not counted towards the gas estimates when running sims, right?

Is that by design?

We've noticed this because we had to update the gas multiplier (adjustment) from 1.2 to 2.0 in our tests.